packages/terra-notification-dialog/tests/jest/NotificationDialog.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import intlContexts from './intl-context-setup';
+import CompleteNotificationDialog from '../../src/terra-dev-site/test/notification-dialog/CompleteNotificationDialog.test';
+import CompleteNotificationDialogWithLongMessage from '../../src/terra-dev-site/test/notification-dialog/CompleteNotificationDialogWithLongMessage.test';
+import NoVariantNotificationDialog from '../../src/terra-dev-site/test/notification-dialog/NoVariantNotificationDialog.test';
+
+it('should shallow a complete notification-dialog', () => {
```

Comment:
It seems there are a lot of prop combinations we haven't tests to ensure the notification dialog renders as expected if a prop is not provided.